static/src/javascripts/projects/common/modules/analytics/interaction-tracking.js

```diff
@@ -1,35 +1,13 @@
 import { mediator } from 'lib/mediator';
 import { storage } from '@guardian/libs';
-import {
-    trackSamePageLinkClick as trackSamePageLinkClickGoogle,
-    trackNonClickInteraction as trackNonClickInteractionGoogle,
-    trackExternalLinkClick as trackExternalLinkClickGoogle,
-    trackSponsorLogoLinkClick as trackSponsorLogoLinkClickGoogle,
-} from 'common/modules/analytics/google';
 import { catchErrorsWithContext } from 'lib/robust';
 
 const NG_STORAGE_KEY = 'gu.analytics.referrerVars';
 let loc = document.location;
 
-
-const isSponsorLogoLinkClick = (target) =>
-    target.hasAttribute('data-sponsor');
-
-// used where we don't have an element to pass as a tag, eg. keyboard interaction
-const trackNonClickInteraction = (actionName) => {
-    trackNonClickInteractionGoogle(actionName);
-};
-
-const trackSamePageLinkClick = (spec) => {
-    // Do not perform a same-page track link when there isn't a tag.
-    if (spec.tag) {
-        trackSamePageLinkClickGoogle(spec.target, spec.tag);
-    }
-};
-
 const trackInternalLinkClick = (spec) => {
     // Store in session storage.
-    // GA and Omniture will both pick it up on next page load,
```

Comment:
Not sure if other code needs to be deleted in this file. I kept the code as it is used for Omniture as well unless someone thinks I should do something differently.

Reply:
I don't this we use Omniture anymore, but in any case it's best to remove in a separate PR.